src/Interpreters/Cache/LRUFileCachePriority.cpp

```diff
@@ -6,6 +6,7 @@
 #include <Common/CurrentThread.h>
 #include <Common/logger_useful.h>
 #include <Common/randomSeed.h>
+#include "base/defines.h"
```

Comment:
```suggestion
#include <base/defines.h>
```
What for is it needed? I do not see